bin/dev

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+docker compose up -d db db-test redis sidekiq
```

Comment:
Maybe check if a user has v2 installed otherwise use `docker-compose`? Some systems (like arch) do not have v2 installed without specifically installing the binary for it

Reply:
Resolved by just using `docker-compose` so everyone will manage to run it. We'll switch to `docker compose` when the adoption rate is higher.